rma_repair/models/repair.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 APSL-Nagarro Antoni Marroig
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class RepairOrder(models.Model):
+    _inherit = "repair.order"
+
+    rma_ids = fields.One2many(
+        comodel_name="rma",
+        inverse_name="repair_id",
+        string="RMAs",
+    )
+
+    def action_view_repair_rma(self):
+        return {
```

Comment:
Why limiting the view to the first RMA ?

Reply:
I try to link one rma with one repair and i think this is the right way

Reply:
What do you think about it?